template/5-mvp.md

```diff
@@ -2,68 +2,92 @@
+#### User Stories for Local Explorers:
+1. "As a local explorer, I want to discover and hidden gems in my city by exploring posts from other residents."
+2. "As a local resident, I want to create local challenges that encourage people to explore lesser-known areas of my city."
 
-*Identify and prioritise the key features required.*
-*Justify the importance of each feature.*
-
-- User Authentication: Necessary for security and personalized user experiences.
-- Post Creation: Central to the app's purpose of sharing and discovering locations.
-- Location-Based Feed: Key to the unique value proposition of Proxima, encouraging physical exploration.
-- Commenting System: Enhances user interaction and engagement.
-- Offline Mode: Ensures usability even without internet access, improving user retention.
-- Voting System: Helps surface the best content and enhances user experience.
-- Posts and comments management: Allows users to track and manage their activity.
-- Centauri Points: Adds a competitive and rewarding element, increasing user engagement.
-- Challenges: Drives user activity and exploration, fostering a dynamic user community.
-- Map Integration: Allows users to view posts on a map and navigate to specific locations.
+#### Key Features:
+- **User Authentication:** Necessary for security and personalized user experiences.
+- **Post Creation and Location-Based Feed:** The core functionality of sharing and discovering places.
+- **Interaction Tools (Comments and Voting):** Enhances user interaction and engagement.
+- **Centauri Points and Challenges:** Motivate continued app usage by creating challenges with rewards in form of points
+- **Ranking System:** Encourages competition and engagement by displaying user rankings based on Centauri points.
+- **Map:** Allows users to navigate to post locations and to visualize nearby activities.
+- **Posts and comments management:** Allows users to track and manage their activity.
+- **Offline Mode:** Ensures usability even without internet access, improving user retention.
 
 
 ## Success Criteria
 
-*How will you evaluate the success of the MVP?*
-*Metrics include user penetration, quality / satisfaction.*
-*If applicable, progress in discussions with ecosystem partners / investors / customers.*
+#### Evaluation Metrics:
+- **User Adoption:** Measured by download numbers and active accounts.
+- **Engagement Metrics:** Frequency of app usage, interactions per user, and completion of challenges.
+- **User Feedback:** Direct feedback and satisfaction surveys to gauge user experience.
+- **Partnerships and Investor Interest:** Progress in securing support and funding based on app traction and user growth.
 
-- User Adoption and Engagement
-- Active User Retention
-- User Satisfaction and Feedback
-- Partnerships and Investment
+#### Benchmarking:
+- **User Adoption:** Targeting 1,000 downloads within the first month.
```

Comment:
I believe we could be a little more ambitious here.
Given that we focuses on Switzerland for the launch, after looking I believe that Switzerland has about 276 500 students. I believe we could target around 20,000 downloads the first month. 

This represents about 7 % of the students. I think that this can be reasonnable given that our application is pretty general.

I'de like to get your thoughts on this ?

Reply:
I think it would be too ambitious in only 1 month, it goes on very fast.
But it really depends on our marketing strategies.
If I recall we wanted to target only EPFL students for the launch, which is only 10'000.
We could aim for 3000 maybe but anything bigger than 50% sounds unrealistic to me.

Reply:
If we only target EPFL for launch then it's all good to me :+1: